tests/reconfiguration.py

```diff
@@ -33,11 +34,15 @@ def count_nodes(configs, network):
 def check_can_progress(node, timeout=3):
     with node.client() as c:
         r = c.get("/node/commit")
+        original_seqno = ccf.clients.parse_tx_id(r.body.json()["transaction_id"])[1]
```

Comment:
I think that if we're going to have a `parse_tx_id` that's longer than `split('.')`, we should have it return a TxID with named fields